packages/bot-analytics/prisma/schema.prisma

```diff
@@ -99,17 +103,21 @@ model AccountActivity {
 model AggregatedLiquidityByMarket {
   id Int @id @unique @default(autoincrement())
```

Comment:
Could this id be created with account + market?

Reply:
Indeed

Reply:
A bit long but if you think it's better I will implement it